init-tools.sh

```diff
@@ -164,6 +164,8 @@ if [ ! -e $__INIT_TOOLS_DONE_MARKER ]; then
     echo Copying supplemental overrides from Tools-Override.
     cp $__scriptpath/Tools-Override/* $__scriptpath/Tools
 
+    Tools-Override/crossgen.sh $__scriptpath/Tools
```

Comment:
I used `--skipcrossgen` in [cibuild.sh](https://github.com/dotnet/roslyn/blob/master/cibuild.sh#L13) in Roslyn, but I think flowing parameters across all the build scripts into init-tools.sh might be a little bit of a pain.  Maybe just use the environment variable `BUILDTOOLS_SKIP_CROSSGEN`?